airframe-ulid/src/main/scala/wvlet/airframe/ulid/ULID.scala

```diff
@@ -329,3 +329,7 @@ object ULID {
   }
 
 }
+
+case class ResourceID(prefix: String, ulid: ULID, delimiter: Char = ':') {
```

Comment:
ResourceID is a too specific name. As a library, having a generic name is preferred. 

- PrefixedULID 
- PULID 
- etc. 

What do you think?

Reply:
I prefer PrefixedULID. In my opinion,  it is difficult to quickly understand what `PULID` is.